### PR TITLE
update to appwrite 1.3.0

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: CHANGELOG.md
Comment (lohanidamodar) [inline CHANGELOG.md:1]: Something is wrong with Changelog and it's not updated as well. Think this was supposed to be merged as I released Flutter/Dart SDK for 1.2.x later https://github.com/appwrite/appwrite/pull/5259

File: docs/examples/account/update-password.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ void main() { // Init SDK
   ;
 
   Future result = account.updatePassword(
-    password: 'password',
+    password: '',
```
Comment (lohanidamodar) [inline docs/examples/account/update-password.md:14]: was this intended?

File: docs/examples/databases/update-boolean-attribute.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+    .setKey('919c2d18fb5d4...a2ae413da83346ad2') // Your secret API key
+  ;
+
+  Future result = databases.updateBooleanAttribute(
+    databaseId: '[DATABASE_ID]',
+    collectionId: '[COLLECTION_ID]',
+    key: '',
```
Comment (lohanidamodar) [inline docs/examples/databases/update-boolean-attribute.md:16]: is this supposed to be empty?

File: docs/examples/teams/create-membership.md [hunk 3]
```diff
@@ -12,7 +12,6 @@ void main() { // Init SDK
 
   Future result = teams.createMembership(
     teamId: '[TEAM_ID]',
-    email: 'email@example.com',
     roles: [],
     url: 'https://example.com',
```
Comment (lohanidamodar) [inline docs/examples/teams/create-membership.md:16]: this example looks weird, because none of the parameters are required 🤔

**Review by lohanidamodar — APPROVED** [review]

**Review by lohanidamodar — APPROVED** [review]

File: docs/examples/teams/create-membership.md (hunk 3 above)
Comment (christyjacob4) [reply docs/examples/teams/create-membership.md:16]: Yeap I've created a task for it

**Pull request merged**